pkg/hub/options/options.go

```diff
@@ -22,6 +22,7 @@ import (
 
 	utilerrors "k8s.io/apimachinery/pkg/util/errors"
 	"k8s.io/apiserver/pkg/admission/plugin/namespace/lifecycle"
+	validatingadmissionpolicy "k8s.io/apiserver/pkg/admission/plugin/policy/validating"
```

Comment:
For 0.17, Clusternet used k8s 1.29 as dependencies. Thus here should be `k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy`.